src/Button/Button.jsx

```diff
@@ -1,27 +1,44 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
-import css from '@styled-system/css'
 import { compose, layout, position, space, variant } from 'styled-system'
 import propTypes from '@styled-system/prop-types'
 
 import { ActivityIndicator } from '@ivoryio/kogaio'
 import { Typography } from '../Typography'
-
-import variants, { spinnerVariants } from './variants'
+import { Flex, Space } from '../'
+import variants, { iconVariants, spinnerVariants } from './variants'
+import { Icon } from '../Icon'
+import colors from '../assets/colors'
 
 const Button = ({
-  children,
-  disabled,
-  icon,
-  id,
-  loading,
-  onClick,
-  type,
-  variant,
-  ...rest
-}) => (
-  <StyledButton
+                  children,
+                  disabled,
+                  icon,
+                  id,
+                  loading,
+                  onClick,
+                  type,
+                  variant,
+                  iconPosition,
+                  ...rest
+                }) => {
```

Comment:
Please use the prettier/webstorm config, this indentation is not according to our rules.